### PR TITLE
Color scheme update to follow styleguide

### LINKED ISSUE #27: Enforce set style specification

### PR DESCRIPTION
**Pull request by thomssonkarl**
Fixes #27 

Implemented the set color scheme for the diagram. 
Further improvements to me made: 
- Add support for text being outside start and valueBox nodes instead of inside. 
- Notes SVG:s should probably be refactored and turned into properly made SVG:s instead of just screenshots.

### CODE REVIEW

File: src/components/nestedList.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const NestedList = () => {
 
   useEffect(() => {
     setDefaultOpenSettings()
-  }, [])
```
Comment (JoXW100) [inline src/components/nestedList.js:70]: `useEffect` should have an empty dependency array here to reduce hov often it runs. it should even warn about it

File: src/styles/diagramNodeGroup.module.css [hunk 2]
```diff
@@ -24,8 +24,9 @@
     padding: 10px;
     width: 370px;
     height: 100px;
-    background-color: rgba(224, 151, 125, 0.5);
+    background-color: #F0CCC1;
```
Comment (JoXW100) [inline src/styles/diagramNodeGroup.module.css:27]: To better organize the colors used, a global color palette might be interesting, see: https://stackoverflow.com/questions/15831657/creating-css-global-variables-stylesheet-theme-management

File: src/components/diagram/customNodes/diagramNodeHolder/roleSphere.js [hunk 3]
```diff
@@ -1,15 +1,59 @@
...
+      case 'KG':
+      case 'P':
+      case 'R':
+      case 'O':
+      case 'L':
+      case 'EV':
+      case 'An/Iva':
+        return '#FF0102'
+      default:
+        return '#808080'
```
Comment (JoXW100) [inline src/components/diagram/customNodes/diagramNodeHolder/roleSphere.js:35]: What colors are these? could be useful to name them

**Review by JoXW100 — CHANGES_REQUESTED** [review]
The color palette i purposed is optional but could make things easier in the future.

File: src/components/nestedList.js (hunk 1 above)
Comment (thomssonkarl) [reply src/components/nestedList.js:70]: It warns about it being empty and suggests to remove the array

File: src/components/diagram/customNodes/diagramNodeHolder/roleSphere.js (hunk 3 above)
Comment (thomssonkarl) [reply src/components/diagram/customNodes/diagramNodeHolder/roleSphere.js:35]: What do you mean by this? They are the colors for the roles

**Review by JoXW100 — APPROVED** [review]

**Review by mackan1410 — APPROVED** [review]

**Pull request merged**